samples/node/text_generation.js

```diff
@@ -176,7 +170,7 @@ async function textGenMultimodalMultiImagePromptStreaming() {
   // Print text as it comes in.
   for await (const chunk of result.stream) {
     const chunkText = chunk.text();
-    console.log(chunkText);
+    process.stdout.write(chunkText);
```

Comment:
For my own understanding: Why do we need to do `process.stdout.write` instead of `console.log` in streaming samples?

Reply:
Ah- sorry, I see your comment about this preventing line breaks.

Reply:
There's a line break after every console.log but there isn't after process.stdout.write. So this makes the streaming continue the line if the chunk broke in the middle of a sentence or something, instead of starting the next chunk on a newline, so the final output looks normal.